internal/cmd/connector_test.go

```diff
@@ -198,7 +200,21 @@ func TestConnector_Run_Kubernetes(t *testing.T) {
 	})
 	assert.DeepEqual(t, fakeKube.writes, expectedWrites, cmpKubeRequest)
 
-	// TODO: check proxy is listening
+	<-opts.Addrs.Available
```

Comment:
This is necessary to prevent a data race. The `opts.Addrs` are set by a goroutine, so we need to block until they are set.